app/models/task/Task.scala

```diff
@@ -331,201 +372,3 @@ object TaskSQLDAO extends SQLDAO[TaskSQL, TasksRow, Tasks] {
 
 }
 
-
-
-
-
-
-class info(message: String) extends scala.annotation.StaticAnnotation
-
-case class Task(
-                 @info("Reference to task type") _taskType: BSONObjectID,
```

Comment:
Valid question. I decided to remove them because
 - I found that they did not add valuable information that was not already in the variable names. 
 - They were not for documenting the Frontend-API (just internally the backend case class).
 - They existed just for Task, and adding them for the other collections for consistency would be extra work
 - They tended to get outdated

Reply:
Seems legit, let's remove them.